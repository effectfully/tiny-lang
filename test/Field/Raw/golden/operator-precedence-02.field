# Testing binding of []
    a + b [c] == a + (b [c])
and a + b [c] <= (a + b) [c]
