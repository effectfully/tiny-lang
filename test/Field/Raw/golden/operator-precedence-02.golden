EAppBinOp And (EAppBinOp FEq (EAppBinOp Add (EVar (Var "a")) (EAppBinOp BAt (EVar (Var "b")) (EVar (Var "c")))) (EAppBinOp Add (EVar (Var "a")) (EAppBinOp BAt (EVar (Var "b")) (EVar (Var "c"))))) (EAppBinOp FLe (EAppBinOp Add (EVar (Var "a")) (EAppBinOp BAt (EVar (Var "b")) (EVar (Var "c")))) (EAppBinOp BAt (EAppBinOp Add (EVar (Var "a")) (EVar (Var "b"))) (EVar (Var "c"))))